### PR TITLE
FY 03 home page

### LINKED ISSUE #5: FY-03-HomePage

### PR DESCRIPTION
**Pull request by sadathshariff**
1. Added HomePage
2. Implemented  Custom Hook for calling the API
3. added categories
4. refactored some styles

### CODE REVIEW

**Review by nishasen — APPROVED** [review]
Code looks clean and good, liked your spinner and well organised folder structure.

File: src/components/Banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <img src={Banner2} className="hero" />
+    </div>
+
+    // <Fade className="hero_container" {...properties}>
+    //   {fadeImages.map((fadeImage, index) => (
+    //     <div className="hero_container" key={index}>
+    //       <img src={fadeImage} className="hero" />
+    //     </div>
+    //   ))}
+    // </Fade>
```
Comment (aishwarya-bm) [inline src/components/Banner/Banner.jsx:30]: Remove comments if not required

File: src/components/Banner/Banner.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Banner1,
+    Banner2,
+    Banner4,
+    // "https://images.unsplash.com/photo-1543584756-8f40a802e14f?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwxfDB8MXxyYW5kb218MHx8fHx8fHx8MTY1MDI5Mjk5Ng&ixlib=rb-1.2.1&q=80&utm_campaign=api-credit&utm_medium=referral&utm_source=unsplash_source&w=1080",
+    // "https://images.unsplash.com/photo-1579600161224-cac5a2971069?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MXwxfDB8MXxhbGx8fHx8fHx8fA&ixlib=rb-1.2.1&q=80&w=1080&utm_source=unsplash_source&utm_medium=referral&utm_campaign=api-credit",
+  ];
+
+  return (
+    <div className="hero_container">
+      <img src={Banner2} className="hero" />
```
Comment (aishwarya-bm) [inline src/components/Banner/Banner.jsx]: Give alt

**Review by aishwarya-bm — APPROVED** [review]
Code looks clean and readable, good naming convention. Left minor suggestions

File: src/components/Banner/Banner.jsx (hunk 2 above)
Comment (sadathshariff) [reply src/components/Banner/Banner.jsx]: ThankYou for pointing it, will add

**Pull request merged**